files/mote-updater.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=cache updater for mote
+After=network.target
+Documentation=https://github.com/fedora-infra/mote
+
+[Service]
+ExecStart=/usr/bin/mote-updater
```

Comment:
Where in the spec does the fedmsg-consumer file get copied to /usr/bin/mote-updater?